AutomatedTesting/Gem/PythonTests/EditorPythonTestTools/editor_python_test_tools/prefab_utils.py

```diff
@@ -27,17 +27,7 @@
 import azlmbr.math as math
 import azlmbr.prefab as prefab
 import editor_python_test_tools.pyside_utils as pyside_utils
-
-
-def get_prefab_file_path(prefab_path):
-    if not path.isabs(prefab_path):
-        prefab_path = path.join(general.get_file_alias("@projectroot@"), prefab_path)
-    
-    # Append prefab if it doesn't contain .prefab on it
-    name, ext = path.splitext(prefab_path)
-    if ext != ".prefab":
-        prefab_path = name + ".prefab"
-    return prefab_path
```

Comment:
Removing this breaks prefab loading from file (not from memory).
Why was this removed?